CONTRIBUTING.rst

```diff
@@ -60,14 +58,29 @@ To add or update test data:
       from rsciio.tests.registry_utils import update_registry
 
       update_registry()
-    
+  
+   On windows, you can use the :ref:`pre-commit hooks <pre-commit-hooks>` from the pull
```

Comment:
Here it says 'on windows', below it says 'unix only' - which one is right?

Reply:
I tweaked the wording to make it less ambiguous. On windows, it is possible to use the pre-commit hook through pre-commit.ci to run on the pull request by requesting it in a comment (on the pull request).